File: python/stdlib/boolean/const.py
try:
    from ...stdlib import Node as base
except ValueError:
    from stdlib import Node as base


class Node(base.Node):
    def __init__(self, verbose, args):
        super(Node, self).__init__("Const", "boolean.const", True, {},
                                   {"result": "Boolean"}, "Output the arg.", verbose, True, False)
        self.args = args

    def tick(self, value):
        return {"result": self.args}

File: lua/stdlib/boolean/const.lua
local myNode = {}
function myNode.init(node)
  node.tick = function(value)
    result = {}
    result.result = node.args.value
    return result
  end
end

function myNode.spec(node)
  node.name = "Const"
  node.outputs.result = "Boolean"
  node.args.value = true
  node.desc = "Return a const boolean"
end

return myNode
